profiles/standard/standard.install

```diff
@@ -401,3 +401,20 @@ function standard_install() {
   config_set('system.core', 'preprocess_js', '1');
   config_set('system.core', 'cache', '1');
 }
+
+/**
+ * Set home page menu to render as a drop-down menu.
+ */
+function standard_update_1000() {
+  if ($layout = layout_load('home')) {
```

Comment:
ugh, this stupid $config object is impossible to work with... are you sure I can't use $layout?